imports/blocks/give/index.js

```diff
@@ -13,41 +14,44 @@ import Loading from "./Loading";
 import Err from "./Err";
 import Success from "./Success";
 
-const mapQueriesToProps = () => ({
-  data: {
-    query: gql`
-      query GetCheckoutData($state: Int!, $country: Int!) {
-        states: definedValues(id: $state, all: true) {
-          name: description, value, id, _id
-        }
-        countries: definedValues(id: $country, all: true) {
-          name: description, value, id, _id
-        }
-        person: currentPerson {
-          firstName
-          nickName
-          lastName
-          email
-          campus { name, id: entityId }
-          home { street1, street2, city, state, zip, country }
-        }
-        savedPayments {
-          name, id: entityId, date,
-          payment { accountNumber, paymentType }
-        }
-        campuses { name, id: entityId }
-      }
-    `,
+const CHECKOUT_QUERY = gql`
+  query GetCheckoutData($state: Int!, $country: Int!) {
+    states: definedValues(id: $state, all: true) {
+      name: description, value, id, _id
+    }
+    countries: definedValues(id: $country, all: true) {
+      name: description, value, id, _id
+    }
+    person: currentPerson {
+      firstName
+      nickName
+      lastName
+      email
+      campus { name, id: entityId }
+      home { street1, street2, city, state, zip, country }
+    }
+    savedPayments {
+      name, id: entityId, date,
+      payment { accountNumber, paymentType }
+    }
+    campuses { name, id: entityId }
+  }
+`;
+
+const withCheckout = graphql(CHECKOUT_QUERY, {
+  options: () => ({
     variables: { state: 28, country: 45 },
```

Comment:
I shortened this, but it still needed to drop down a line to avoid a lint error for max length.